cypress/e2e/test/Login.cy.js

```diff
@@ -0,0 +1,32 @@
+import { visitHomePage } from "../utils";
+import * as login from "../pages/login";
+
+describe("Login Functionality", () => {
+  beforeEach(() => {
+    // visit and sign in function call
+    visitHomePage();
+  });
+  it("should login successfully with valid credentials", () => {
+    // login with valid credenctials
+    login.AuthenticateUser("rihef83861@godsigma.com", "passwordtest");
+  });
+  // it("should show error message for invalid credentials", () => {
```

Comment:
Hi @monikasingh949, Can you please remove commented code or if it's working and you want to keep it please un comment.